unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -17,67 +17,49 @@ public class SceneController : IMessageProcessHandler, IMessageQueueHandler
     {
         public static bool VERBOSE = false;
 
-        //======================================================================
-
-        #region PROJECT_ENTRYPOINT
-
-        //======================================================================
-        private EntryPoint_World worldEntryPoint;
-
         public DCLComponentFactory componentFactory => Main.i.componentFactory;
 
         public bool enabled = true;
-        public bool initialized = false;
+
+        private Coroutine deferredDecodingCoroutine;
 
         public void Initialize()
         {
-            if (initialized)
-                return;
-
             sceneSortDirty = true;
             positionDirty = true;
             lastSortFrame = 0;
             enabled = true;
-            initialized = true;
 
-            Environment.i.debugController.OnDebugModeSet += OnDebugModeSet;
+            Environment.i.platform.debugController.OnDebugModeSet += OnDebugModeSet;
 
             // We trigger the Decentraland logic once SceneController has been instanced and is ready to act.
             WebInterface.StartDecentraland();
 
-            Environment.i.parcelScenesCleaner.Start();
-
             if (deferredMessagesDecoding) // We should be able to delete this code
-                CoroutineStarter.Start(DeferredDecoding()); //
+                deferredDecodingCoroutine = CoroutineStarter.Start(DeferredDecoding()); //
 
             DCLCharacterController.OnCharacterMoved += SetPositionDirty;
 
             CommonScriptableObjects.sceneID.OnChange += OnCurrentSceneIdChange;
 
-            //TODO(Brian): Move those suscriptions elsewhere.
-            PoolManager.i.OnGet -= Environment.i.physicsSyncController.MarkDirty;
-            PoolManager.i.OnGet += Environment.i.physicsSyncController.MarkDirty;
-            PoolManager.i.OnGet -= Environment.i.cullingController.objectsTracker.MarkDirty;
-            PoolManager.i.OnGet += Environment.i.cullingController.objectsTracker.MarkDirty;
-
-#if !UNITY_EDITOR
-            worldEntryPoint = new EntryPoint_World(this); // independent subsystem => put at entrypoint but not at environment
-#endif
-
-            // TODO(Brian): This should be fixed when we do the proper initialization layer
-            if (!EnvironmentSettings.RUNNING_TESTS)
-            {
-                Environment.i.cullingController.Start();
-            }
+            //TODO(Brian): Move those subscriptions elsewhere.
```

Comment:
Only a doubt about this `TODO` comment: Why should these subscriptions be moved from the `Initialize()`?

Reply:
Those subscriptions are related to the `physicsSyncController` and `cullingController`. Having them there is a heavy violation on encapsulation, the subscriptions should be performed in the same systems that use it.